specs/registration_spec.js

```diff
@@ -0,0 +1,19 @@
+let indexPage = require ("../page_objects/index_page")
```

Comment:
File name is `registration_spec.js` but inside we have `unsuccessful login` test. Looks like registration test is missed